02/app.js

```diff
@@ -0,0 +1,109 @@
+const form = document.querySelector("form");
+
+const isEmailValid = (form) => {
+  const loginElem = form.elements.login;
+
+  if (typeof loginElem !== "undefined") {
+    const email = loginElem.value;
+    const emailRe = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
+
+    return emailRe.test(email);
+  }
+
+  return false;
+};
+
+const isPasswordValid = (passwordElem) => {
```

Comment:
Tu już lepiej, ale znów sugeruję już przekazać wartość.
PS. Jak funkcja strzałkowa ma 1 parametr to nie piszemy nawiasów okrągłych.